docs/x509/verification.rst

```diff
@@ -21,3 +21,44 @@ chain building, etc.
 
     :param certs: A list of one or more :class:`~cryptography.x509.Certificate`
         instances.
+
+.. class:: Subject
+
+    .. versionadded:: 42.0.0
+
+    Type alias: A union of all subject types supported:
+    :class:`cryptography.x509.general_name.DNSName`,
+    :class:`cryptography.x509.general_name.IPAddress`.
+
+
+.. class:: PolicyBuilder
+
+    .. versionadded:: 42.0.0
+
+    A PolicyBuilder provides a builder-style interface for constructing a
+    Policy.
+
+    .. classmethod:: webpki
+
+        Creates a new :class:`PolicyBuilder` with defaults for the Web PKI,
+        i.e. client verification of web server TLS certificates.
+
```

Comment:
Let's not include this yet.

Reply:
https://github.com/pyca/cryptography/pull/9601/commits/2d857e258ac7e942f277dc4e7b5ec8d363a0fd72